src/status_im/chat/events.cljs

```diff
@@ -105,8 +105,11 @@
   [re-frame/trim-v]
   (fn [{:keys [db] :as cofx} [envelope-hash status]]
     (let [{:keys [chat-id message-id]} (get-in db [:transport/message-envelopes envelope-hash])
-          message (get-in db [:chats chat-id :messages message-id])]
-      (models.message/update-message-status message status cofx))))
+          message (get-in db [:chats chat-id :messages message-id])
+          {:keys [fcm-token]} (get-in db [:contacts/contacts chat-id])]
+      (cond-> (models.message/update-message-status message status cofx)
```

Comment:
it would be more readable to use merge-fx, the condition can be put into send-push-notification function

Reply:
changed to `merge-fx`